### PR TITLE
Graph cors

### PR DESCRIPTION
**Pull request by JacqueGM**
enable regex CORS in graphnode

### CODE REVIEW

**Review by llunaCreixent — COMMENTED** [review]
It doesn't work from the testing domain produced by this testing branch: https://github.com/CirclesUBI/circles-myxogastria/pull/649
```
https://deploy-preview-649.preview-review.circles.garden/send/0x9f5FF18027ADBB65A53086CDc09D12ce463daE0B?a=1000&n=return
```
When I click "send" I see the error in the relayer request for estimation.

**Comment by JacqueGM** [comment]
This PR is only for the CORS for graph as it says in the name and the description :) for the relayer is different ^^

**Review by llunaCreixent — APPROVED** [review]
looks good then for graph CORS, still we need to create the PR for the relayer

File: helm/circles-graphprotocol/values.yaml [hunk 1]
```diff
@@ -58,4 +58,6 @@ CORS_allowed:
   - "https://preview-dev.circles.garden"
   - "https://preview-product.circles.garden"
   - "https://preview-review.circles.garden"
-  - "https://preview-testing.circles.garden"
+
+CORS_allowed_regex:
+  - http[s]?:\/\/.*\.circles\.garden.*
```
[the remark this answers is not in the document]
Comment (juanenrisley) [reply helm/circles-graphprotocol/values.yaml:63]: I do not know a solution yet for this kind of problem. So at the moment, let's continue and we can revisit it in the future.

**Review by juanenrisley — DISMISSED** [review]

**Review by juanenrisley — APPROVED** [review]
👍🏿

**Pull request merged**